### DISCUSSION TITLE
is it possible to get the all date between a week based on the week number?

### DISCUSSION THREAD

**Discussion by Kellermaan** [SilentVoid13/Templater#558] [Help]
Hi, I'm using the periodical note.
And it can generate the file with the correct week number as its name.
I want to ask that Is it possible to get the all date between a week based on the week number in the file name?

Thank!

**Comment by AB1908** [comment]
See if this helps: https://stackoverflow.com/questions/16590500/calculate-date-from-week-number-in-javascript

You'd have to use a templater script with some of those ideas. Let me know if you need help with it.
